ci/tasks/light-aws/test-drivers.sh

```
@@ -0,0 +1,63 @@
+
+echo "Downloading machine image"
+export MACHINE_IMAGE_PATH=${tmp_dir}/image.iso
+export MACHINE_IMAGE_FORMAT="RAW"
+wget -O ${MACHINE_IMAGE_PATH} http://tinycorelinux.net/7.x/x86_64/archive/7.1/TinyCorePure64-7.1.iso
+
+echo "Running driver tests"
+
+pushd "${REPO_PARENT}/builder-src" > /dev/null
+  # Run all driver specs in parallel to reduce test time
+  spec_count="$(grep "It(" -r driver | wc -l)"
+  go run github.com/onsi/ginkgo/v2/ginkgo -nodes ${spec_count} -r driver
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)